src/containers/search/filterSelector/style.scss

```diff
@@ -44,6 +44,14 @@
 	margin-right: 0.25rem;
 }
 
+ul.filterList {
+  list-style-type: none;
+}
+
+ul.filterList li {
+  list-style-type: none;
+}
+
```

Comment:
Do we need custom classes for this? Would Bootstrap's `list-unstyled` (https://getbootstrap.com/docs/4.4/content/typography/#unstyled) work?

Reply:
That's what I tried first, but sadly it doesn't indent